pyatv/mrp/messages.py

```diff
@@ -42,6 +42,36 @@ def device_information(name, identifier):
     return message
 
 
+def device_information_update(name, identifier, logical_device_count):
```

Comment:
Maybe we can do some code sharing with `device_information` here? I generally not a fan of passing bool-flags, but maybe we can live with that here (i.e. pick correct message type from a flag)?